svg/global_attributes.json

```diff
@@ -794,12 +794,12 @@
           "spec_url": "https://svgwg.org/svg2-draft/struct.html#ConditionalProcessingSystemLanguageAttribute",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
This is defined in https://svgwg.org/svg2-draft/types.html#InterfaceSVGTests but https://mdn-bcd-collector.gooborg.com/tests/svg/global_attributes/systemLanguage is invalid because it's looking for the property on a generic `SVGElement`, but but the IDL attribute is on `SVGAnimationElement` and `SVGGraphicsElement`